user-interface/frontend/themes/datamanager/components/all.css

```diff
@@ -1,8 +1,332 @@
+}
+
+.heading-4 {
+  font-size: var(--lumo-font-size-l);
+  font-weight: var(--font-weight-semi-bold);
+}
+
+.heading-5 {
+  font-size: var(--lumo-font-size-m);
+  font-weight: var(--font-weight-semi-bold);
+}
+
+.heading-6 {
+  font-size: var(--lumo-font-size-xs);
+  font-weight: var(--font-weight-semi-bold);
+}
+
+
+/*
+Body text
+ */
+
+.normal-body-text {
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--lumo-line-height-m);
+  font-weight: var(--font-weight-regular);
+}
+
+.small-body-text {
+  font-size: var(--lumo-font-size-s);
+  line-height: var(--lumo-line-height-s);
+  font-weight: var(--font-weight-regular);
+}
+
+.extra-small-body-text {
+  font-size: var(--lumo-font-size-xs);
+  line-height: var(--lumo-line-height-xs);
+  font-weight: var(--font-weight-regular);
+}
+
+.field-label-text {
+  font-size: var(--lumo-font-size-s);
+  line-height: var(--lumo-line-height-xs);
+  font-weight: var(--font-weight-medium);
+}
+
+.input-field-text {
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--lumo-line-height-s);
+  font-weight: var(--font-weight-medium);
+}
+
+.list-item-text {
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--spacing-07);
+  font-weight: var(--font-weight-regular);
+}
+
+.button-text {
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--lumo-line-height-xs);
+  font-weight: var(--font-weight-medium);
+}
+
+.button-text-primary {
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--lumo-line-height-xs);
+  font-weight: var(--font-weight-semi-bold);
+}
+
+.text-margin-bottom-03 {
+  margin-bottom: var(--spacing-03);
+}
+
+.text-margin-bottom-04 {
+  margin-bottom: var(--spacing-04);
+}
+
+
+/***************************
+Padding & gaps
+***************************/
+
+.padding-left-right-04 {
+  padding-left: var(--spacing-04);
+  padding-right: var(--spacing-04);
+}
+
+.padding-left-right-05 {
+  padding-left: var(--spacing-05);
+  padding-right: var(--spacing-05);
+}
+
+.padding-left-right-06 {
+  padding-left: var(--spacing-06);
+  padding-right: var(--spacing-06);
+}
+
+.padding-left-right-07 {
+  padding-left: var(--spacing-07);
+  padding-right: var(--spacing-07);
+}
+
+.padding-top-bottom-04 {
+  padding-top: var(--spacing-04);
+  padding-bottom: var(--spacing-04);
+}
+
+.padding-top-bottom-05 {
+  padding-top: var(--spacing-05);
+  padding-bottom: var(--spacing-05);
+}
+
+.padding-top-bottom-06 {
+  padding-top: var(--spacing-06);
+  padding-bottom: var(--spacing-06);
+}
+
+.padding-top-bottom-07 {
+  padding-top: var(--spacing-07);
+  padding-bottom: var(--spacing-07);
+}
+
+.gap-04 {
+  gap: var(--spacing-04);
+}
+
+.gap-05 {
+  gap: var(--spacing-05);
+}
+
+.gap-06 {
+  gap: var(--spacing-06);
+}
+
+.gap-07 {
+  gap: var(--spacing-07);
+}
+
+
+/***************************
+Dialogs
+***************************/
+
+/*
+Sections
+ */
+.dialog-section {
+  display: flex;
+  flex-direction: column;
+}
+
+/*
+Dialog flavours
+ */
+.simple-dialog {}
```

Comment:
Where would other components go? In contrast to the previous sections of this css, you go away from the functional style like `gap-07` to component specific layouting `.simple-dialog`.